src/locale/locales/de/messages.po

```diff
@@ -192,37 +187,33 @@ msgstr ""
 msgid "Appeal Content Warning"
 msgstr ""
 
-#: src/view/com/modals/AppealLabel.tsx:65
-#~ msgid "Appeal Decision"
-#~ msgstr "Berufungsentscheidung"
-
 #: src/view/com/util/moderation/LabelInfo.tsx:52
 msgid "Appeal this decision"
 msgstr "Einspruch gegen diese Entscheidung"
 
 #: src/view/com/util/moderation/LabelInfo.tsx:56
 msgid "Appeal this decision."
-msgstr "Einspruch gegen diese Entscheidung"
+msgstr "Einspruch gegen diese Entscheidung."
 
 #: src/view/screens/Settings.tsx:432
 msgid "Appearance"
-msgstr "Aussehen"
+msgstr "Erscheinungsbild"
 
 #: src/view/screens/AppPasswords.tsx:223
 msgid "Are you sure you want to delete the app password \"{name}\"?"
-msgstr "Sind Sie sicher, dass Sie das App-Passwort \"{name}\" löschen möchten?"
+msgstr "Bist du sicher, dass du das App-Passwort \"{name}\" löschen möchtest?"
 
 #: src/view/com/composer/Composer.tsx:142
 msgid "Are you sure you'd like to discard this draft?"
-msgstr "Sind Sie sicher, dass Sie diesen Entwurf verwerfen möchten?"
+msgstr "Bist du sicher, dass du diesen Entwurf verwerfen möchtest?"
 
 #: src/view/screens/ProfileList.tsx:352
 msgid "Are you sure?"
-msgstr "Sind Sie sicher?"
+msgstr "Bist du sicher?"
 
 #: src/view/com/util/forms/PostDropdownBtn.tsx:219
 msgid "Are you sure? This cannot be undone."
-msgstr "Sind Sie sicher? Dies kann nicht rückgängig gemacht werden"
+msgstr "Bist du sicher? Dies kann nicht rückgängig gemacht werden."
```

Comment:
Maybe "Diese Entscheidung" instead of just "Dies"? Sounds more natural to me.

Reply:
This appears if you want to delete your post after publication. It's below "Delete this post?", so I think the short version should suffice.

Reply:
Could also be "Diese Aktion kann nicht rückgängig gemacht werden"